packages/buttons/src/elements/Anchor.spec.tsx

```diff
@@ -15,4 +15,26 @@ describe('Anchor', () => {
 
     expect(container.querySelector('svg')).not.toBeNull();
   });
+
+  it('renders link security attributes while external', () => {
+    const { getByText } = render(<Anchor isExternal>link</Anchor>);
+
+    const anchor = getByText('link');
+
+    expect(anchor).toHaveAttribute('target', '_blank');
```

Comment:
I’m not sure that we want all external links to open in new windows. Do we? 

In my understanding, the `rel` precautions only apply to new windows. Then we'd want to auto-set them if and only if the user of this component already wanted `target=_blank`.

That said, if it were a product recommendation at Zendesk for our products to always open third-party links in a new window, well, that is fair enough and in that case this line LGTM!

Reply:
Good call out @alanhogan. It is Garden's recommendation to always open third-arty links in a new window; we anticipate this mimics user intention most of the time. Feels like a good, smart default (while still allowing for customized use cases where we might not want this).

Reply:
I think the `external` icon is most often used to denote that a link is opening in a new context (tab in this case), not that the link is going to a third-party.

Another option would be to only add the `rel` attribute if `target="_blank"` is provided. This would be more specific, but starts to feel complicated for consumers to figure out.

Reply:
But, to @allisonacs 's point, most third-party links are opened in a new tab from this guidance. Even for first-party properties this seems like a good benefit and is customizable if tweaks are necessary.